.github/workflows/ci-submodule.yml

```diff
@@ -25,10 +25,13 @@ jobs:
       build_result: ${{ steps.build.outcome }}
     strategy:
       matrix:
-        target: ['armv7a7-imx6ull-evk', 'armv7a9-zynq7000-qemu', 'armv7a9-zynq7000-zedboard', 'armv7a9-zynq7000-zturn', 'armv7m4-stm32l4x6-nucleo', 'armv7m7-imxrt106x-evk', 'armv7m7-imxrt117x-evk', 'host-generic-pc', 'ia32-generic-pc', 'ia32-generic-qemu', 'riscv64-generic-qemu', 'riscv64-generic-spike']
+        target: ['armv7a7-imx6ull-evk', 'armv7a9-zynq7000-zedboard', 'armv7a9-zynq7000-zturn', 'armv7m4-stm32l4x6-nucleo', 'armv7m7-imxrt106x-evk', 'armv7m7-imxrt117x-evk', 'host-generic-pc', 'ia32-generic-pc', 'ia32-generic-qemu', 'riscv64-generic-qemu', 'riscv64-generic-spike']
+        additional_params: ['']
```

Comment:
Do we need this empty `additional_params`?

Reply:
Oh, that's right. It isn't necessary here, now it's updated.

Reply:
Do we need to remove `armv7a9-zynq7000-qemu` from target list? `ia32-generic-qemu` is there and work wells with additional `syspage` param

Reply:
I think it could be specified in `target`. Maybe it's a better way, the change was applied in the last force push.

Reply:
And last note, why we removed quotation marks? `''`